pkg/builder/builder.go

```diff
@@ -140,7 +145,36 @@ func buildConfigVolumeMount(vdb *vapi.VerticaDB) corev1.VolumeMount {
 	}
 }
 
-// buildVolumeMounts returns the volume mounts to include in the sts pod spec
+// buildServerVolumeMounts returns the volume mounts to include
+// in the server container
+func buildServerVolumeMounts(vdb *vapi.VerticaDB) []corev1.VolumeMount {
+	volMnts := buildVolumeMounts(vdb)
```

Comment:
We mount a subpath to scrutinize at /tmp/scrutinize. I think we should do that only in the nma container. We should not run scrutinize in the server container. We might not even have the NMA TLS certs mounted, which scrutinize uses.

Reply:
I missed that part, thanks!